File: eform-client/src/app/plugins/modules/items-group-planning-pn/models/list/uploaded-data.model.ts
export class UploadedDatasModel {
  total: number;
  uploadedDatas: Array<UploadedDataModel> = [];
}

export class UploadedDataModel {
  id: number;
  itemCaseId: number;
  checkSum: string;
  extension: string;
  currentFile: string;
  uploaderType: string;
  fileLocation: string;
  fileName: string;
}

File: eFormAPI/Plugins/ItemsGroupPlanning.Pn/ItemsGroupPlanning.Pn/Infrastructure/Models/UploadedDataModel.cs
namespace ItemsGroupPlanning.Pn.Infrastructure.Models
{
    public class UploadedDataModel
    {
        public int Id { get; set; }
        
        public int ItemCaseId { get; set; }
        
        public string Checksum { get; set; }

        public string Extension { get; set; }

        public string CurrentFile { get; set; }

        public string UploaderType { get; set; }

        public string FileLocation { get; set; }

        public string FileName { get; set; }

    }
}